client/my-sites/email/inbox/index.jsx

```diff
@@ -30,6 +31,19 @@ const NoAccessCard = () => {
 	);
 };
 
+const NotSupportedOnP2Card = () => {
+	const translate = useTranslate();
+
+	return (
+		<Main>
+			<EmptyContent
+				title={ translate( 'Inbox is not supported on P2 sites' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'P2+ is supported only on P2 sites.' )` *ES Score: 6*

Reply:
I think this string is missing a trailing dot @olaseni .